### DISCUSSION TITLE
Tooling to send articles to podcastfy and publish result to own podcast station

### DISCUSSION THREAD

**Discussion by robert-elles** [souzatharsis/podcastfy#203] [Ideas]
Hi,
I am wondering if there already exists a project making use of podcastfy such that I could send a link of an article to it, the article gets turned into a podcast episode by podcastfy and the episode gets published via a podcast feed which I can subscribe to from a podcast app on my phone.
If not I want to try building something very simple myself. I am thinking maybe just a telegram bot or web app that I can send the links to. It should be easily hostable on a home server.
Any suggestion on projects that do this or parts of it and how to quickly put something together very appreciated.

**Comment by souzatharsis** [comment]
I recommend the following:


   - OpenNotebook <https://www.open-notebook.ai/>
   - SurfSense <https://www.surfsense.net/>
   - OpenPod <https://openpod.fly.dev/>


Best Regards,

--
Thársis
souzatharsis.com <http://www.souzatharsis.com/>
<http://linkedin.com/in/tharsissouza>
